src/expr/src/vector_op/trigonometric.rs

```diff
@@ -56,9 +56,55 @@ pub fn atan2_f64(input_x: F64, input_y: F64) -> F64 {
     input_x.0.atan2(input_y.0).into()
 }
 
+#[function("sind(float64) -> float64")]
+pub fn sind_f64(input: F64) -> F64 {
+    f64::sin(f64::to_radians(input.0)).into()
+}
```

Comment:
The PostgreSQL implementation is nontrivial, as it is [intended](https://www.postgresql.org/docs/current/functions-math.html) to be more accurate than conversion first.
> Another way to work with angles measured in degrees is to use the unit transformation functions radians() and degrees() shown earlier. However, using the degree-based trigonometric functions is preferred, as that way avoids round-off error for special cases such as sind(30).

For reference, see [this](https://github.com/postgres/postgres/blob/REL_15_2/src/backend/utils/adt/float.c#L2444).

Reply:
Thank you very much for pointing that out. Sorry, I did not have time to continue working on this. I will try to introduce the function one by one whenever I have time for it. My first short at `Sind`: https://github.com/risingwavelabs/risingwave/pull/9056

Reply:
`Cosd`: https://github.com/risingwavelabs/risingwave/pull/9064/